app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import datetime
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
According to the checklist, it is a good practice to use annotations for all your variables. In this case, you missed the annotation for the 'cafe' variable. It is a good practice to annotate all variables, not only the ones with basic types. This helps to understand the expected type of the variable and can prevent possible bugs.